ludwig/encoders/bag_encoders.py

```diff
@@ -76,11 +79,12 @@ def __init__(
             embeddings_on_cpu=embeddings_on_cpu,
             dropout=dropout,
             embedding_initializer=weights_initializer,
-            embedding_regularizer=weights_regularizer
+            # embedding_regularizer=weights_regularizer
```

Comment:
Could you remove the commented out code here?

Reply:
Done.